test/unit/network_utils_spec.js

```diff
@@ -0,0 +1,178 @@
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createResponseStatusError, validateRangeRequestCapabilities,
+  validateResponseStatus
+} from '../../src/display/network_utils';
+import {
+  MissingPDFException, UnexpectedResponseException
+} from '../../src/shared/util';
+
+describe('network_utils', function() {
+  describe('validateRangeRequestCapabilities', function() {
+    const defaultValues = {
+      allowRangeRequests: false,
+      suggestedLength: undefined,
+    };
+
+    it('rejects range chunk sizes that are not strictly positive', function() {
+      expect(function() {
+        validateRangeRequestCapabilities({ rangeChunkSize: 0, });
+      }).toThrow(new Error());
```

Comment:
This is an existing issue, but can you please add a suitable error message to the `assert` in the core code, such that we can test that the "correct" error is thrown here?

Reply:
Done in the new commit.

Thank you for the review, and I wish you a merry Christmas too!